llm/wit/golem-llm.wit

```diff
@@ -230,16 +218,30 @@ interface llm {
     provider-error-json: option<string>,
   }
 
-  // --- Chat Response Variants ---
+  // --- Chat Response ---
 
-  /// One resulting event in an LLM conversation
-  variant chat-event {
-    /// A response in the conversation
-    message(complete-response),
-    /// A request for calling one or more tools
-    tool-request(list<tool-call>),
-    /// An error in the conversation
-    error(error),
+  /// Response from an LLM
+  record response {
+    /// Response ID
+    id: string,
+    /// Result contents
+    content: list<content-part>,
+    /// Tool call requests
+    tool-calls: list<tool-call>,
+    /// Response metadata
+    metadata: response-metadata,
+  }
+
+  // --- Chat event  ---
+
+  /// Chat events that can happen during a chat session
+  variant event {
+    /// Message asked by the user
+    message(message),
```

Comment:
This is not clear to me (maybe just the comment). This the simple text answer case? Or the user input? Or how is it different form `response`.

Reply:
Ok the `send` function suggests that this is the "input".

Reply:
message is added by the user, response is by the agent